### PR TITLE
README and dependencies

### PR DESCRIPTION
**Pull request by mpcsh**
- Made the example code in the README actually execute
- Moved `istanbul` (coverage tool) to `devDependencies` where it belongs
- Cut down on line lengths in `stream.iced`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,26 +28,15 @@ var es = new saltpack.stream.EncryptStream({
...
-file.on('close', () -> ds.end())
+es.on('error', function (err) { throw err; })
+ds.on('error', function (err) { throw err; })
+
+//roundtrip encrypt/decrypt from stdin to stdout
+process.stdin.pipe(es)
+//{Encrypt,Decrypt}Streams don't yet know how to accept pipes, so we have to pipe to
+//their .first_stream.
+es.pipe(ds.first_stream);
+ds.pipe(process.stdout)
```
Comment (oconnor663) [inline README.md:41]: Let's add some description somewhere about what this is doing. ("Encrypting a message and then decrypting it.")

**Review by oconnor663 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mpcsh) [reply README.md:41]: Wait, do you need to refresh your diff? Line 34 is `//roundtrip encrypt/decrypt from stdin to stdout`. Or did you want me to edit that?

Comment (oconnor663) [reply README.md:41]: Ah good point. I did miss that. I was also hoping for something a littler higher-level. Maybe at the top of the script, or before it starts? Basically, the reader has no idea what this is doing until they get to line 34. We need to give them something like "here's what you're about to learn".

Comment (mpcsh) [reply README.md:41]: oh, true. added be234f9

**Pull request merged**